### ISSUE TITLE
mambaforge is now miniforge  

### ISSUE THREAD

**Issue by mikemhenry** [openforcefield/openff-docs#31]
See:
https://github.com/conda-forge/miniforge#whats-the-difference-between-mambaforge-and-miniforge

So [links like this on the install page](https://docs.openforcefield.org/en/latest/install.html):
https://github.com/conda-forge/miniforge#user-content-mambaforge

don't "work" (they don't 404 but since that section doesn't exist anymore, they don't direct the user where we want them) anymore.

I think if we update references to mambaforge to miniforge then we will be good.

**Comment by Yoshanuikabundi** [comment]
Good call. I'm very tempted to recommend Micromamba actually, I use it locally and not having a base environment is great (though I'm annoyed that it doesn't alphabetize the list of packages it's going to install like Mamba does!). Do you have any thoughts about that?

**Comment by mattwthompson** [comment]
+1 for recommending `micromamba` to developers

-1 for recommending `micromamba` everywhere; people have already been confused enough by `mamba` and incurring another switch will result in many developer hours shifted towards user support for what amounts roughly to feature parity and a smaller executable

**Comment by mikemhenry** [comment]
If `mamba` didn't allow a `base` env (or at least didn't make is special) that would cover the biggest footgun that `micromamba` protects users from, but I do agree it is just different enough to cause issues

**Comment by mattwthompson** [comment]
I think this is done, did not look to see which change it came through with

<img width="1220" height="1185" alt="Image" src="https://github.com/user-attachments/assets/35cad05f-0a57-4530-959d-514e968bd609" />
